man/systemd.resource-control.xml

```diff
@@ -264,8 +264,14 @@
         <term><varname>MemoryMin=<replaceable>bytes</replaceable></varname></term>
 
         <listitem>
-          <para>Specify the memory usage protection of the executed processes in this unit. If the memory usages of
-          this unit and all its ancestors are below their minimum boundaries, this unit's memory won't be reclaimed.</para>
+          <para>Specify the memory usage protection of the executed processes in this unit.
+          When reclaiming memory, the unit is treated as if it was using less memory resulting in memory
+          to be preferentially reclaimed from unprotected units.
+          This is a strong protection, the OOM killer will be invoked if there is no other reclaimable memory.
+          A <varname>MemoryLow=</varname> allocation also needs to be assigned to ancestor slice units
+          as this allocation will be distributed across all children.
+          Any <varname>MemoryLow=</varname> allocation that is not explicitly distributed will protect
+          all children equally.</para>
```

Comment:
Hmm, is "equally" the best word here? The kernel docs say "This allows protecting entire subtrees from one another, while retaining free competition within those subtrees". So the kernel does not care if the distribution is "equal".

Reply:
You are right, "equal" doesn't capture it correctly.

I wanted to explain it from a slightly different perspective. Maybe something closer to the kernel wording would work.

Maybe: `Any <varname>MemoryLow=</varname> allocation that is not explicitly distributed to specific children is used to create a shared protection for all children. As this is a shared protection, the children will freely compete for the memory.`